codesrc/src/main/java/com/onehealth/core/exceptions/InValidRequestException.java

```diff
@@ -0,0 +1,5 @@
+package com.onehealth.core.exceptions;
+
+public class InValidRequestException extends RuntimeException {
```

Comment:
Add status code for all type of exceptions that will flow to UI to identity it in UI